graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndicesResource.java

```diff
@@ -104,10 +105,10 @@ public IndexInfo single(@ApiParam(name = "index") @PathParam("index") String ind
 
     @GET
     @Timed
-    @ApiOperation(value = "Get information of all indices managed by Graylog and their shards.")
+    @ApiOperation(value = "Get information of all open indices managed by Graylog and their shards.")
     @RequiresPermissions(RestPermissions.INDICES_READ)
     @Produces(MediaType.APPLICATION_JSON)
-    public AllIndicesInfo all() {
+    public OpenIndicesInfo open() {
```

Comment:
I think this resource should be named similar to the other resources in this class, i. e. "/open" &rarr; "/system/indexer/indices/open".